wandb/sdk/wandb_run.py

```diff
@@ -1046,7 +1108,8 @@ def display(self, height: int = 420, hidden: bool = False) -> bool:
             wandb.termwarn(".display() only works in jupyter environments")
             return False
 
-    @Attach._attach
+    # @_run_decorator._noop
```

Comment:
Have you tried uncommenting these, does it break anything?

Reply:
yeah... it works! 
@raubitsj prefers not to add it to all the functions, since he is worried that it will break users code that worked although it shouldn't have (log already had this logic before)